mmv1/third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -321,6 +321,18 @@ Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 * `enable_batching` - (Optional) Defaults to true. If false, disables global
 batching and each request is sent normally.
 
+## User Agent Extension
```

Comment:
```suggestion
---
```

Reply:
Let's keep this as a entry in the advanced settings, rather than a new header itself! (This page looks significantly different than the live one atm- should catch up in a release or two)